onnx/backend/test/case/node/__init__.py

```diff
@@ -37,6 +38,13 @@ def function_expand_helper(node,  # type: NodeProto
         io_names_map[function_proto.input[idx]] = node.input[idx] \
             if idx in range(len(node.input)) else ""
 
+        # Some nodes in the function body might contain subgraphs, and those could depend on the
```

Comment:
You are right, the existing code doesn't seem to handle subgraphs in nodes in the function body. The right fix would be to recursively traverse the subgraph and rename references to function input names (just like it is done for the top-level). The fix below will likely fail if we have two calls to the function which are both expanded, since the names will clash.

Reply:
I've now implemented the subgraph traverse algorithm you describe. Can you please take a look?